src/main/java/org/jitsi/videobridge/IceUdpTransportManager.java

```diff
@@ -46,7 +46,7 @@
      * The name default of the single <tt>IceStream</tt> that this
      * <tt>TransportManager</tt> will create/use.
      */
-    private static final String DEFAULT_ICE_STREAM_NAME = "stream";
+    protected static final String DEFAULT_ICE_STREAM_NAME = "stream";
```

Comment:
it doesn't look like we use this in the subclass, do we?  we just use the stream directly